modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -9214,48 +9231,60 @@ private void _testDeleteObjectEntryWithAccountEntryRestricted2(
 	}
 
 	private void _testDeleteRelatedObjectEntry(
+			ObjectDefinition objectDefinitionAA, ObjectEntry objectEntryA,
+			ObjectEntry objectEntryB,
+			ObjectField objectRelationshipA_AAObjectField2,
+			ObjectField objectRelationshipB_AAObjectField2,
+			ObjectRelationship objectRelationshipA_AA,
```

Comment:
`objectRelationshipA_AA` should come before `objectRelationshipA_AAObjectField2`